docs/requirements.txt

```diff
@@ -1,6 +1,7 @@
-docutils==0.17.1
-jinja2<3.1
 m2r
+Sphinx==6.2.1
+docutils==0.18.1
 mistune==0.8.4
-sphinx-rtd-theme
-sphinx==3.0.3
+sphinx-rtd-theme==1.2.2
+pyyaml==6.0.1
```

Comment:
Stupid dask

Reply:
I considered upgrading Dask, but since that actually modifies running code libraries, it seemed more risky. This feels silly, but at least this requirements file is only used in the context of docs-building